### PR TITLE
Update Appendix L

### PR DESCRIPTION
**Pull request by StateCourtsePDJoyce**
APPENDIX L - PRE-ACTION PROTOCOL FOR BUSINESS-TO-BUSINESS DEBT CLAIMS

### CODE REVIEW

File: Appendices.md [hunk 1]
```diff
@@ -1380,3 +1380,166 @@ is less than $20,000 (excluding interest, if any), the Court will in general, aw
 ## Footnotes
 
 [^17]: Please refer to Order 92, Rule 1 of the Rules of Court (Cap. 322 R 5).
+
+
+## Appendix L - PRE-ACTION PROTOCOL FOR BUSINESS-TO-BUSINESS DEBT CLAIMS
```
Comment (LoneRifle) [inline Appendices.md]:
```suggestion
## Appendix L - Pre-Action Protocol for Business-to-Business Debt Claims
```

File: Appendices.md [hunk 2]
```diff
@@ -1380,3 +1380,166 @@ is less than $20,000 (excluding interest, if any), the Court will in general, aw
...
+
+> (a)	a sole proprietorship;
+
+> (b)	a partnership;
+
+> (c)	a company, including a foreign company;
+
+> (d)	a limited partnership; and
+
+> (e)	a limited liability partnership;
```
Comment (LoneRifle) [inline Appendices.md]:
```suggestion
> (e)	a limited liability partnership.
```

**Review by LoneRifle — COMMENTED** [review]
lgtm otherwise

**Comment by StateCourtsePDJoyce** [comment]
Hi Lone,

Can you call me at <masked>? One amendment has been updated wrongly and
need you to ignore it. Thanks.

Regards,
Joyce

**Comment by LoneRifle** [comment]
Hi @StateCourtsePDJoyce - you are free to make further edits to the document! On this page, click files changed, click the three dots on the `Appendices.md` file that is displayed, and then choose `Edit file`

**Pull request merged**